ce/session.ts

```diff
@@ -120,8 +108,8 @@ export class Session {
     this.setupLogging();
 
     this.homeFolder = this.fileSystem.file(settings['homeFolder']!);
-    this.cache = this.environment['VCPKG_DOWNLOADS'] ? this.parseUri(this.environment['VCPKG_DOWNLOADS']) : this.homeFolder.join('downloads');
-    this.globalConfig = this.homeFolder.join('vcpkg-configuration.global.json');
+    this.cache = this.environment[vcpkgDownloadFolder] ? this.parseUri(this.environment[vcpkgDownloadFolder]!) : this.homeFolder.join('downloads');
```

Comment:
FYI I added a note that vcpkg.exe needs to fill this value in for you in https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems/edit/1444644